app/components/program-year/objective-list.js

```diff
@@ -73,8 +74,6 @@ export default class ProgramYearObjectiveListComponent extends Component {
     const resourcePath = `/programyears/${this.args.programYear.id}/downloadobjectivesmapping`;
     const host = this.iliosConfig.apiHost ?? `${window.location.protocol}//${window.location.host}`;
     const url = host + apiPath + resourcePath;
-    const { saveAs } = yield import('file-saver');
```

Comment:
The issue is that import filesaver returns an `FileSaver.default.saveAs` and not `FileSaver.saveAs`. Check my syntax on the destrucutring, but I think this should be `const { default: saveAs } = yield import('file-saver');`.

Reply:
updated :heavy_check_mark: